src/sparseml/pytorch/torchvision/train.py

```diff
@@ -138,7 +138,7 @@ def train_one_epoch(
                 # Reset ema buffer to keep copying weights during warmup period
                 model_ema.n_averaged.fill_(0)
 
-        acc1, acc5 = utils.accuracy(output, target, topk=(1, 5))
+        acc1, _1, acc5, _5 = utils.accuracy(output, target, topk=(1, 5))
         batch_size = image.shape[0]
         metric_logger.update(loss=loss.item(), lr=optimizer.param_groups[0]["lr"])
         metric_logger.meters["acc1"].update(acc1.item(), n=batch_size)
```

Comment:
Does the new total param need to be added to this & acc5 like in evaluate?

Reply:
For evaluation, it is added since we wish to compute the global accuracy.
For training, global avg doesn't make sense since the model keeps evolving. However, we can include if we wish to maintain uniformity. Let me know if that's preferred and I will update.

Reply:
yeah no we don't want to compute global accuracy during training. Nice to have is the running average across batches to smooth the curve, but i can still live w/o it.